tools/runas-_bios-script

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+_TIMEOUT=60
+
+if [[ "$EUID" -ne 0 ]]; then
+   echo "This utility must be run as root"
+   exit 1
+fi
+
+while getopts t:c:e: name
+do
+  case "$name" in
+  t)
+    _TIMEOUT="$OPTARG";;
+  e)
+    export "$OPTARG";;
+  ?)
+    usage;;
+  esac
+done
+
+shift $(($OPTIND - 1))
+
+# We need to properly quote/escape the command to not mung it.
+COMMAND=
+for i; do
```

Comment:
Something missing ?

Reply:
Not in this case. We want to build `COMMAND` from arguments with properly escaped quotes and `for i;` is a synonym for `for i in "$@";`.